Packs/Jira/Scripts/ScriptJiraListStatus/ScriptJiraListStatus.py

```diff
@@ -0,0 +1,100 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def create_output(error_msg):
+    return {
+        "Type": entryTypes["error"],
+        "ContentsFormat": formats["text"],
+        "Contents": error_msg,
+        "HumanReadable": error_msg,
+        "ReadableContentsFormat": formats["text"],
+    }
+
+
+def execute_xsoar_command(args: Dict[str, Any]) -> List[Dict[str, Any]]:
```

Comment:
Here again - it is more `execute_jira_list_transition_command` than `execute_xsoar_command`.